pkgs/development/rocm-modules/6/llvm/default.nix

```diff
@@ -258,13 +279,8 @@ let
     ]
   );
 in
-rec {
-  inherit (llvmPackagesRocm) libcxx;
-  inherit args;
-  # Pass through original attrs for debugging where non-overridden llvm/clang is getting used
-  # llvm-orig = llvmPackagesRocm.llvm; # nix why-depends --derivation .#rocmPackages.clr .#rocmPackages.llvm.llvm-orig
-  # clang-orig = llvmPackagesRocm.clang; # nix why-depends --derivation .#rocmPackages.clr .#rocmPackages.llvm.clang-orig
-  llvm = llvmPackagesRocm.llvm.overrideAttrs (old: {
+overrideLlvmPackagesRocm (s: {
+  libllvm = (s.prev.libllvm.override { }).overrideAttrs (old: {
```

Comment:
Remind me what the `.override { }` is doing here? I think I knew once, but then forgot. It should of course be a no‐op, so it scares me a little if it’s not – is it related to the rebuild avoidance? Will it start being a no‐op again after we make a follow‐up change?